src/compiler/compile_config.rs

```diff
@@ -1,7 +1,12 @@
 use crate::path::OwnedTargetPath;
-use anymap::AnyMap;
-use std::collections::BTreeSet;
+use std::{
+    any::{Any, TypeId},
+    collections::{BTreeSet, HashMap},
+};
 
+type AnyMap = HashMap<TypeId, Box<dyn Any>>;
+
+#[derive(Default)]
 pub struct CompileConfig {
```

Comment:
Do we have any unit test coverage for this?

Reply:
Good call. I'm not seeing any. I'll add some..